src/feed-store.js

```diff
@@ -306,29 +306,36 @@ export class FeedStore extends EventEmitter {
   /**
    * Creates a ReadableStream from the loaded feeds.
    *
-   * Uses a callback function to return the stream for each feed.
-   * NOTE: If the callback returns `false` it will ignore the feed.
-   *
-   * @param {StreamCallback} [callback] Function to call the feed.createReadStream() for each feed.
+   * @param {Object} [options] Default options for each feed.createReadStream(options).
+   * @param {StreamCallback} [callback] Filter function to return options for each feed.createReadStream(). Returns `undefined` will ignore the feed.
    * @returns {ReadableStream}
    */
-  createReadStream (callback = ({ feed }) => feed.createReadStream()) {
+  createReadStream (options, callback = () => ({})) {
```

Comment:
Prefer for readability to have a const with defaultCallback, but not too important.